### ISSUE TITLE
Error when using `exiftool-vendored` module with Bun

### ISSUE THREAD

**Issue by MarioMH8** [oven-sh/bun#8259]
### What version of Bun is running?

1.0.23+83f2432da

### What platform is your computer?

Darwin 23.2.0 arm64 arm

### What steps can reproduce the bug?

```
bun add exiftool-vendored
```

```typeScript
// script.ts
const exiftool = require("exiftool-vendored").exiftool

exiftool
  .version()
  .then((version) => console.log(`We're running ExifTool v${version}`));
```

```
bun run ./script.ts
```

### What is the expected behavior?

The `exiftool-vendored` module should function without errors, allowing image metadata manipulation.

### What do you see instead?

```
error: Cannot find package "exiftool-vendored.pl" from "./node_modules/exiftool-vendored/dist/FindExiftool.js"
```

**Comment by Electroid** [comment]
Do you see the same issue if you remove your node_modules folder and install with `npm install`?

**Comment by MarioMH8** [comment]
No, with `npm` it works fine.

**Comment by dylan-conway** [comment]
Fixed in bun v1.1.28 with #13848
